lib/endpoints/class-wp-rest-controller.php

```diff
@@ -365,7 +365,7 @@ protected function get_object_type() {
 	 *                           Where as create requests will.
 	 * @return array
 	 */
-	public function get_endpoint_args_for_item_schema( $add_required_flag = true ) {
+	public function get_endpoint_args_for_item_schema( $method = WP_REST_Server::CREATABLE ) {
```

Comment:
Should we maybe rename this to just `get_endpoint_args`, and make `$method` required?